diff --git a/contrib/cmake/FindSDL2.cmake b/contrib/cmake/FindSDL2.cmake
index 05663da..8e2b27b 100644
--- a/contrib/cmake/FindSDL2.cmake
+++ b/contrib/cmake/FindSDL2.cmake
@@ -76,10 +76,10 @@ SET(SDL2_SEARCH_PATHS
 	${SDL2_PATH}
 )
 
-FIND_PATH(SDL2_INCLUDE_DIR SDL2/SDL.h
+FIND_PATH(SDL2_INCLUDE_DIR SDL.h
 	HINTS
 	$ENV{SDL2DIR}
-	PATH_SUFFIXES include
+	PATH_SUFFIXES include/SDL2
 	PATHS ${SDL2_SEARCH_PATHS}
 )
 
